Core/ResponseHandler.php

```diff
@@ -55,7 +58,19 @@ public static function onSuccessResponse($oResponse, $oBackendService, $oOrder)
 
         if (!self::_isPostProcessingAction($oResponse)) {
             self::_updateOrder($oOrder, $oResponse, $oBackendService);
-            $oOrder->sendOrderByEmail();
+
+            try {
+                $oOrder->sendOrderByEmail();
+            } catch (SystemComponentException $exc) {
```

Comment:
we should check that the message contains `getThumbnailUrl `, so we don't just catch all `SystemComponentException`

Reply:
good point 👍